scripts/globals/garrison_data.lua

```diff
@@ -0,0 +1,812 @@
+            {
+            },
+            [2] =
+            {
+            },
+            [3] =
+            {
+            },
+            [4] =
+            {
+            },
+            [5] =
+            {
+            },
+            [6] =
+            {
+            },
+        },
+    },
+    [xi.zone.WEST_SARUTABARUTA] =
+    {
+        itemReq = 1530, -- Seven-Knot Quipu
+        levelCap = 20,
+        npcs = {
+        -- empty filled with dynamic entries
+        },
+        mobs = 17248598,
+        xPos = -0,
+        yPos = -0,
+        zPos = -0,
+        xChange = 0,
+        zChange = 2,
+        xSecondLine = 2,
+        zSecondLine = 0,
+        xThirdLine = 4,
+        zThirdLine = 0,
+        rot = 0,
+        name = "676e7063C", -- gnpc text to hex (rename with proper names later)waveSize = {
+        waveSize = {
+        -- Wave + Number of Parties
+            [1] =
+            {
+            },
+            [2] =
+            {
+            },
+            [3] =
+            {
+            },
+            [4] =
+            {
+            },
+            [5] =
+            {
+            },
+            [6] =
+            {
+            },
+        },
+    },
+    [xi.zone.VALKURM_DUNES] =
+    {
+        itemReq = 1531, -- Galka Fang Sack
+        levelCap = 30,
+        npcs = {
+        -- empty filled with dynamic entries
+        },
+        mobs = 17199593,
+        xPos = 141,
+        yPos = -8,
+        zPos = 87,
+        xChange = -2,
+        zChange = -2,
+        xSecondLine = 2,
+        zSecondLine = 0,
+        xThirdLine = 4,
+        zThirdLine = 0,
+        rot = 32,
+        name = "676e7063C", -- gnpc text to hex (rename with proper names later)
+        waveSize = {
+        -- Wave + Number of Parties
+            [1] =
+            {
+                17199593, -- Goblin Swordmaker
+                17199594 -- Goblin Swordmaker
+            },
+            [2] =
+            {
+                17199593, -- Goblin Swordmaker
+                17199594, -- Goblin Swordmaker
+                17199595, -- Goblin Leecher
+                17199596 -- Goblin Leecher
+            },
+            [3] =
+            {
+                17199593, -- Goblin Swordmaker
+                17199594, -- Goblin Swordmaker
+                17199595, -- Goblin Leecher
+                17199596, -- Goblin Leecher
+                17199597, -- Goblin Gaoler
+                17199598 -- Goblin Gaoler
+            },
+            [4] =
+            {
+                17199593, -- Goblin Swordmaker
+                17199594, -- Goblin Swordmaker
+                17199595, -- Goblin Leecher
+                17199596, -- Goblin Leecher
+                17199597, -- Goblin Gaoler
+                17199598, -- Goblin Gaoler
+                17199599, -- Goblin Gambler
+                17199600 -- Goblin Gambler
+            },
+            [5] =
+            {
+                17199593, -- Goblin Swordmaker
```

Comment:
I'm very against having to store all the IDs and other data for each garrison event in one big file like this. There a couple of ways to  skirt around needing the IDs, but there will need to be some discussion before anything is decided; I'll wait for Zach's input especially on this subject since he had a wip for garrison already.

Reply:
There is no expectation of getting my pushed through was meant to be a draft but i suck at git.

As for the tables of all ids.. Im not a fan either but se's stupid wave logic didnt make it easy... Since its not always first ids that spawn first (fighters in ronfaure are first wave and cursemakers which have lowest IDs dont sure up until they all spawn)

Zachs wave tables kind of work just need a way to increase wave size based on number of alliance

Either way smarter tables are needed

Reply:
gotcha. we'll take what we can of it tho.